src/main/pages/setup-openshift/openshift-admin-guide.adoc

```diff
@@ -244,25 +230,25 @@ The Che Master has three possible states:
 * `PREPARING_TO_SHUTDOWN`
 * `READY_TO_SHUTDOWN`
 
-The `PREPARING_TO_SHUTDOWN` state means that no new workspace startups are allowed. This situation can cause two different results: 
+The `PREPARING_TO_SHUTDOWN` state means that no new workspace startups are allowed. This situation can cause two different results:
 
-* If your infrastructure does not support workspace recovery, all running workspaces are forcibly stopped. 
+* If your infrastructure does not support workspace recovery, all running workspaces are forcibly stopped.
 
 * If your infrastructure does support workspace recovery, any workspaces that are currently starting or stopping is allowed to finish that process. Running workspaces do not stop.
 
-For those that did not stop, automatic fallback to the shutdown with full workspaces stopping will be performed. 
+For those that did not stop, automatic fallback to the shutdown with full workspaces stopping will be performed.
 
 If you want a full shutdown with workspaces stopped, you can request this by using the `shutdown=true` parameter. When preparation process is finished, the `READY_TO_SHUTDOWN` state is set which allows to stop current Che master instance.
 
 [id="che-workspace-termination-grace-period"]
-== Che Workspace Termination Grace Period
+== Che workspace termination grace period
 
 [IMPORTANT]
 ====
-If the `terminationGracePeriodSeconds` have been explicitly set in the Kubernetes/OpenShift recipe, this environment variable does not override the recipe.
+If the `terminationGracePeriodSeconds` variable is explicitly set in the OpenShift recipe, this environment variable does not override the recipe.
 ====
 
-The default grace termination period of Kubernetes/OpenShift workspace’s pods is '0'.  This setting terminates pods almost instantly and significantly decreases the time required for stopping a workspace. 
+The default grace termination period of Kubernetes/OpenShift workspace’s pods is '0'.  This setting terminates pods almost instantly and significantly decreases the time required for stopping a workspace.
```

Comment:
* no slash, please
* no need for possessive in _'period of Kubernetes/OpenShift workspace’s pods'_